core/server/api/v2/utils/serializers/output/all.js

```diff
@@ -0,0 +1,25 @@
+const debug = require('ghost-ignition').debug('api:v2:utils:serializers:output:all');
+const _ = require('lodash');
+
+const removeXBY = (object) => {
+    _.each(object, (value, key) => {
+        // CASE: go deeper
+        if (_.isObject(object[key]) || _.isArray(object[key])) {
```

Comment:
```suggestion
         if (_.isObject(value) || _.isArray(value)) {
```
no need for additional property access here, as we are already iterating over properties :)

Reply:
Yeah true 😎Why is Github adding an indent in the beginning? 🤔

Reply:
Oh dammit :octocat:! literally edited just the value there :laughing:

Reply:
Should i push the change manually? let me know :)

Reply:
Changed both places, but can't commit from the page :upside_down_face: